harnessed_jobs/rtm_aliveness_power_on/v0/ccs_rtm_aliveness_power_on.py

```diff
@@ -0,0 +1,209 @@
+"""
+Power-on aliveness tests script, based on Homer's
+harnessed-jobs/T08/rebalive_power/ccseorebalive_power.py script.
+"""
+import sys
+import time
+from collections import namedtuple
+import logging
+import java.lang
+from org.lsst.ccs.scripting import CCS
+
+CCS.setThrowExceptions(True)
+
+logging.basicConfig(format="%(message)s",
+                    level=logging.DEBUG,
+                    #level=logging.INFO,
+                    stream=sys.stdout)
+logger = logging.getLogger()
+
+class CcsSubsystems(object):
+    def __init__(self, subsystems=None):
+        if subsystems is None:
+            subsystems = dict(ts8='ts8', rebps='ccs-rebps')
+        for key, value in subsystems.items():
+            self.__dict__[key] = CCS.attachSubsystem(value)
+
+ChannelInfo = namedtuple('ChannelInfo', ['reb_ps_channel', 'ts8_mon_chan',
+                                         'low_lim', 'high_lim', 'chkreb'])
+
+def set_monitoring_interval(ts8, period_ms, logger=logger):
+    logger.info("Setting tick and monitoring period to %i ms" % period_ms)
+    command = "change monitor-update taskPeriodMillis %i" % period_ms
+    logger.debug(command)
+    ts8.synchCommand(10, command)
+
+def power_off_rebs(lines=(0, 1, 2)):
+    # Power-off the requested REBs via the specified power-lines.
+    for power_line in lines:
+        command = "setNamedPowerOn %d master False" % power_line
+        logger.debug(command)
+        ccs_sub.rebps.synchCommand(10, command)
+
+def map_power_lines_to_rebs(ccs_sub, ntries=20, wait_between_tries=10,
+                            num_lines=3):
+    """
+    Map power lines to REBs by powering on one at a time for each REB
+    and trying to read the 1-wire ID register.  This function will
+    leave the REBs in a powered-off state.
+    """
+    rebids = ccs_sub.ts8.synchCommand(10, "getREBIds").getResult()
+    rebnames = ccs_sub.ts8.synchCommand(10, "getREBDeviceNames").getResult()
+
+    # Ensure that all of the power-lines to the REBs are off to start.
+    power_off_rebs()
+
+    # Loop over each REB to find the line it uses.
+    power_lines = {}
+    for rebid, rebname in zip(rebids, rebnames):
+        line = 0
+        power_line = None
+        while power_line is None and line < num_lines:
+            if line in power_lines.values():
+                line += 1
+                continue
+            for name in 'master digital analog'.split():
+                ccs_sub.rebps.synchCommand(10, 'setNamedPowerOn %d %s True'
+                                           % (line, name))
+                time.sleep(0.5)
```

Comment:
Since the multiple tries below are to allow for the firmware to load after powering on the digital and analog channels, we should start off with a longer wait time, e.g., call `time.sleep(wait_between_tries)` here instead of just using `0.5`s.